.gitignore

```diff
@@ -1,6 +1,7 @@
 .DS_Store
 node_modules
 dist
-package-lock.json
+website/package-lock.json
```

Comment:
We made this change because:

1. you can't gitignore the package-lock.json because it will prevent `npm ci` from working which means that published versions will not have locked transitive dependencies
2. you do want to gitignore the lockfile for the storybook folder since that's not deployed.